sklearn/svm/_classes.py

```diff
@@ -566,23 +572,23 @@ class SVC(BaseSVC):
         non-trivial. See the section about multi-class classification in the
         SVM section of the User Guide for details.
 
-    coef_ : array, shape = [n_class * (n_class-1) / 2, n_features]
-        Weights assigned to the features (coefficients in the primal
-        problem). This is only available in the case of a linear kernel.
-
-        `coef_` is a readonly property derived from `dual_coef_` and
-        `support_vectors_`.
+    fit_status_ : int
+        0 if correctly fitted, 1 otherwise (will raise warning)
 
     intercept_ : ndarray of shape (n_class * (n_class-1) / 2,)
         Constants in decision function.
 
-    fit_status_ : int
-        0 if correctly fitted, 1 otherwise (will raise warning)
+    n_support_ : array-like, dtype=int32, shape = [n_class]
+        Number of support vectors for each class.
 
-    classes_ : array of shape (n_classes,)
-        The classes labels.
+    support_ : array-like of shape (n_SV)
+        Indices of support vectors.
+
+    support_vectors_ : array-like of shape (n_SV, n_features)
```

Comment:
I would assume that it is actually definitely an `array` and not an `array-like`, because it was produced by the class